composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "dxw/whippet",
     "description": "",
+    "version": "1.0.0",
```

Comment:
https://getcomposer.org/doc/04-schema.md#version

> The version of the package. In most cases this is not required and should be omitted (see below).

Instead, add a `v1.0.0` git tag.

Reply:
Good catch, I've removed that now. I'll tag the merge commit once we've got this merged, that somehow feels more correct to me than adding the tag to the last commit in this MR.